gradebook/src/GradeBook/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+/* 
```

Comment:
Suggest using [XML doc comments](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/) throughout.

Reply:
changed the commenting style

Reply:
XML doc comments are the triple slash ones (`///`).  VS has some convenient tooling around them to make descriptions show up in Intellisense and things like that.  I usually add these to at least major public methods.

Reply:
Oh. Yeah I kinda misread as (//). Let me get this corrected.